code/modules/mob/living/simple_animal/abnormality/he/jangsan.dm

```diff
@@ -0,0 +1,288 @@
+	var/list/stats = list(FORTITUDE_ATTRIBUTE,
+			PRUDENCE_ATTRIBUTE,
+			TEMPERANCE_ATTRIBUTE,
+			JUSTICE_ATTRIBUTE)
+//attack vars
+	var/togglemovement
+	var/bite_cooldown
+	var/bite_cooldown_time = 8 SECONDS
+	var/chase_cooldown
+	var/chase_cooldown_time = 8 SECONDS
+	var/lure_cooldown
+	var/lure_cooldown_time = 120 SECONDS
+
+//speak_list + location + speak_list2
+	var/list/speak_list = list(";Hey guys im at ",
+			";Over here at ", ";Im in ")
+	var/list/speak_list2 = list(", let's have a pizza party!",
+			", i'll protect you!", ", let's work together!")
+
+/mob/living/simple_animal/hostile/abnormality/jangsan/Initialize()
+	. = ..()
+	AbnoRadio()
+	RegisterSignal(SSdcs, COMSIG_GLOB_MOB_DEATH, .proc/on_mob_death) // Hell
+
+/mob/living/simple_animal/hostile/abnormality/jangsan/Destroy()
+	UnregisterSignal(SSdcs, COMSIG_GLOB_MOB_DEATH)
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/jangsan/Move()
+	if (!togglemovement)
+		return FALSE
+	..()
+
+/mob/living/simple_animal/hostile/abnormality/jangsan/update_icon_state()
+	if(status_flags & GODMODE)	// Not breaching
+		icon_state = initial(icon)
+	else
+		icon_state = icon_aggro
+
+/mob/living/simple_animal/hostile/abnormality/jangsan/Moved()
+	. = ..()
+	if(!(status_flags & GODMODE)) // Whitaker nerf
+		playsound(get_turf(src), 'sound/abnormalities/bigbird/step.ogg', 50, 1)
+
+/mob/living/simple_animal/hostile/abnormality/jangsan/CanAttack(atom/the_target)
+	if(ishuman(the_target))
+		if(bite_cooldown > world.time)
+			return FALSE
+		var/mob/living/carbon/human/H = the_target
+		var/obj/item/bodypart/head/head = H.get_bodypart("head")
+		if(!istype(head)) // You, I'm afraid, are headless
+			return FALSE
+	togglemovement = TRUE
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/jangsan/AttackingTarget()
+	killCheck(target)
+	icon_state = icon_aggro
+	return ..()
+
+//on-kill effect
+/obj/effect/halo
+	name = "halo"
+	desc = "Poor guy."
+	icon = 'ModularTegustation/Teguicons/tegu_effects.dmi'
+	icon_state = "jangsan_kill"
+	move_force = INFINITY
+	pull_force = INFINITY
+	generic_canpass = FALSE
+	movement_type = PHASING | FLYING
+	layer = POINT_LAYER	//High enough that you can see it if you look up.
+
+/obj/effect/halo/Initialize()
+	..()
+	animate(src, pixel_x = 0, pixel_z = 16, time = 3 SECONDS)
+	QDEL_IN(src, 30 SECONDS)
+
+/mob/living/simple_animal/hostile/abnormality/jangsan/proc/lure()
+	var/myarea
+	if(lure_cooldown > world.time)
+		return
+	lure_cooldown = world.time + lure_cooldown_time
+	var/list/Players = list()
+	for (var/mob/living/carbon/human/H in GLOB.human_list)
+		if(H.z != z) // Not on our level
+			continue
+		if(H.stat == DEAD) // No dead people
+			continue
+		Players += H
+
+	if(!Players.len)
+		name = pick("Unassuming Friendly Guy","Zeta 123","Bong Bong","John Lobotomy")
+	else
+		var/Sucker = pick(Players)
+		name = "[Sucker]"
+	playsound(get_turf(src), 'sound/abnormalities/scaredycat/catgrunt.ogg', 50, 1, 2)
```

Comment:
Re-using sounds of other abnos is bad.
It's not **necessary** that you find new sounds, but that'd be preferable.